libsqlite3-sys/build.rs

```diff
@@ -167,7 +167,14 @@ mod build_bundled {
                         }
                     }
                     Some(openssl_dir) => {
-                        let lib_dir = lib_dir.unwrap_or_else(|| openssl_dir.join("lib"));
+                        let lib_dir = lib_dir.unwrap_or_else(|| {
+                            // OpenSSL 3.0 now puts it's libraries in lib64/ by default,
+                            // check for both it and lib/.
+                            if openssl_dir.join("lib64").exists() {
+                                return openssl_dir.join("lib64");
```

Comment:
Should depend on `target`, no ?

Reply:
I think not, for some architectures, OpenSSL version 3.x is installed under the lib64 directory.

This approach is also in sync with [rust-openssl](https://github.com/sfackler/rust-openssl/blob/master/openssl-sys/build/find_normal.rs#L20).

Reply:
But `rust-openssl` uses both `lib64` **and** `lib`.

Reply:
Yes, I've now fully aligned its logic with that of openssl.

Reply:
Thanks.